src/node/entities.h

```diff
@@ -66,6 +66,8 @@ namespace ccf
     // Nodes identities and allowed code ids
     static constexpr auto NODES = "public:ccf.gov.nodes.info";
     static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
+    static constexpr auto NETWORK_CONFIGURATIONS =
+      "public:ccf.gov.network_configurations";
```

Comment:
`public:ccf.gov.network.configurations` or `public:ccf.gov.nodes.configurations` maybe?

Reply:
Both good suggestions. Are there any conventions for table names? I'm sort of inclined to pick the shortest one :)

Reply:
The conventions are: lower-case, "ccf.gov." for all tables written to by governance directly or indirectly and deemed important for audit purposes. Nesting is fine, but avoid tables named "foo.bar.baz" if there is a table "foo.bar", ie. no table name should be the prefix of another.

That's about it I think. We try to avoid underscores/multiple words where we can, but it's not always possible, there is precedent for it.

Reply:
Done, I went with `public:ccf.gov.network.configurations`, but I'm not emotionally attached to it, so feel free to change it if you think of a better name :)